library/ReplicatedTableEntity.cs

```diff
@@ -100,24 +101,112 @@ public ReplicatedTableEntity(string partitionKey, string rowKey)
             this.RowKey = rowKey;
         }
 
+        public ReplicatedTableEntity(ReplicatedTableEntity entity)
```

Comment:
These 2 new constructors seem to be used only incase of test, if the usage is only expected to be test adding a comment would help